app/Services/UserService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Services;
+
+class UserService extends Service {
+    
+    public function createUser($data = array()) {
+        $db = $this->Db;
+        
+        $username = $data['username'];
+        $email = $data['email'];
+        
+        $query = $db->prepare("INSERT INTO users VALUES (NULL, ?, ?, NOW())");
+        $query->bindParam(1, $username, \PDO::PARAM_STR);
+        $query->bindParam(2, $email, \PDO::PARAM_STR);
+        if(!$query->execute()) {
+            throw new Exception("Something wrong with your query");
```

Comment:
This is not good practice to throw Exception with message 'Something wrong..', it doesn't help to figure out what exactly was wrong, please use $query->errorInfo() method in order to provide some useful message.